localization/languages/hu.json

```diff
@@ -69,10 +69,10 @@
     "newTask": "Új feladat", //"new" is a verb - it is used for a button at the bottom of the task overlay
     "defaultTaskName": "Feladat %n", //this is the name used for newly-created tasks; %n is replaced with a number ("task 1", "task 2", etc)
     "taskDeleteWarning": {
-      "unsafeHTML": null //missing translation
+      "unsafeHTML": "Nem biztonságos HTML fájl" //missing translation
```

Comment:
This should be translated based on the English version:
```
    "taskDeleteWarning": {
      "unsafeHTML": "Task deleted. <a>Undo?</a>"
    },
```

(`unsafeHTML` is just a marker that this translation string has HTML inside it; it isn't displayed to the user)